docs/extensions/working_with_extensions.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: docu
+title: Working with Extensions
+---
+
+## Downloading Extensions Directly from S3
+
+Downloading an extension directly could be helpful when building a lambda or container that uses DuckDB.
+DuckDB extensions are stored in public S3 buckets, but the directory structure of those buckets is not searchable. 
+As a result, a direct URL to the file must be used. 
+To directly download an extension file, use the following format:  
+
+```text
+http://extensions.duckdb.org/v{release_version_number}/{platform_name}/{extension_name}.duckdb_extension.gz
+```
+
+For example:
+
+```text
+http://extensions.duckdb.org/v{{ site.currentduckdbversion }}/windows_amd64/json.duckdb_extension.gz
+```
+
+The list of supported platforms may increase over time, but the current list of platforms includes:
```

Comment:
@szarnyasg are we able to check this list is up-to-date per what is in the bucket?